travertino/pyproject.toml

```diff
@@ -0,0 +1,90 @@
+[build-system]
+requires = [
+    "setuptools==75.6.0",
+    "setuptools_scm==8.1.0",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+dynamic = ["version"]
+name = "travertino"
+description = "A set of constants and base classes for describing user interface layouts."
+readme = "README.rst"
+requires-python = ">= 3.9"
+license.text = "New BSD"
+authors = [
+    {name="Russell Keith-Magee", email="russell@keith-magee.com"},
+]
+maintainers = [
+    {name="BeeWare Team", email="team@beeware.org"},
+]
+keywords = [
+    "css",
+    "box model",
+    "layout",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: User Interfaces",
+]
+
+[project.optional-dependencies]
+# Extras used by developers *of* Toga are pinned to specific versions to
```

Comment:
Not sure what has happened here - these look like the same dependencies (and docstring) for Toga core. It makes sense why typing-extensions and coverage is needed; but why pytest-asyncio and pytest-freezer?